src/models/RekognitionModel.ts

```diff
@@ -13,8 +11,17 @@ export default class RekognitionModel extends BaseModel {
   }
 
   detectFaces (image: string) {
-    return mockResponse;
-    //this.rekognition.detectFaces();
+    const params = {
+      Image: {
+        S3Object: {
+          Bucket: 'images',
```

Comment:
hmmmmmm remind me to check how we are pasing the bucket name... cause this should definitely NOT be hardcoded